industry_benchmarks/utils/fix_networks.py

```diff
@@ -167,14 +168,20 @@ def _check_and_deduplicate_transforms(
 
     for t_name, t_list in transforms_dict.items():
         if len(t_list) != 3:
-            # TODO: we can turn this into a warning message if it's too painful
-            errmsg = (
-                f"Too few transformations found for {t_name} "
-                "this indicates a partially completed set of results. "
-                "Please ensure that your input network is finished and "
-                "any reproducible partial failures have been removed."
-            )
-            raise ValueError(errmsg)
+            # print a message if we want to allow missing repeats
+            if allow_missing:
+                errmsg = (f"Too few transformations found for {t_name}"
+                          f"this indicates a partially complete set of results."
+                          f"This edge will be ignored.")
```

Comment:
It looks like that if the `allow_missing = True`, an edge that has missing data (e.g. only 2 out of 3 repeats) would be treated as if it completed successfully (they are added to the `transform_list`. I think it might be valid to treat them as successful edges, but then the print statement should be edited to reflect them. Maybe even encourage the user to check if those calculations are still running or need to be resubmitted?

Reply:
Good point I think I have been a little inconsistent with what a successful edge is, I think for the sake of the benchmark it should be all 3 repeats of both legs have finished which is easy to check and doesn't complicate downstream analysis, so I'll update this to match the print and ignore the edge.